spec/controllers/hq/users_controller_spec.rb

```diff
@@ -1,25 +1,12 @@
 require 'rails_helper'
-class DummyusersController < Hq::UsersController; end
 
-describe Hq::UsersController, type: :controller do
-  describe 'authentication' do
-    controller DummyusersController do
-      def foobar_action
-        render nothing: true
-      end
-    end
-
-    it 'checks for admin_user session' do
-      routes.draw { get 'foobar_action', to: 'dummyusers#foobar_action' }
-      expect(controller).to be_a_kind_of(Hq::UsersController)
-      expect_any_instance_of(Hq::UsersController).to receive :authenticate_admin_user!
-      get :foobar_action
-    end
+RSpec.describe Hq::UsersController, type: :controller do
+  specify 'authentication' do
+    expect(Hq::UsersController.superclass).to eq HqController
   end
 end
-  
-describe Hq::UsersController, type: :controller do
 
+RSpec.describe Hq::UsersController, type: :controller do
```

Comment:
If we ever turn off the Rails monkey-patching to increase app performance with `config.expose_dsl_globally= false` then this will be needed.  `RSpec.describe` does everything `describe` does, plus more, and (I think) it is more readable what is going on.